pkgs/applications/networking/insync/v3.nix

```diff
@@ -0,0 +1,77 @@
+{ stdenv
+, lib
+, fetchurl
+, makeWrapper
+, dpkg
+, glibc
+, glib
+, libxcb
+, libGL
+, nss
+, libthai
+, wayland
+, alsaLib
+, qt5
```

Comment:
Please pass in the qt packages one by one and use the wrapper callPackage to easily change the qt version used.

Reply:
OK - but this package may be sensitive to the particular version of qt since it's a binary distribution.

Reply:
done